src/args.rs

```diff
@@ -78,17 +78,17 @@ pub(crate) fn clap_app(tmp_dir: &str) -> clap::App {
         .arg(
             Arg::with_name(FTPS_CERTS_FILE)
                 .long("ftps-certs-file")
-                .value_name("PKCS12_ARCHIVE")
-                .help("Sets the path the the certificates used for TLS security/FTPS")
+                .value_name("PEM_FILE")
+                .help("Sets the path the the certificates used for TLS security")
                 .env("UNFTP_FTPS_CERTS_FILE")
                 .takes_value(true),
         )
         .arg(
-            Arg::with_name(FTPS_CERTS_PASSWORD)
-                .long("ftps-certs-password")
-                .value_name("PASSWORD")
-                .help("Sets password used to unlock the archive specified in ftps-certs-file")
-                .env("UNFTP_FTPS_CERTS_PASSWORD")
+            Arg::with_name(FTPS_KEY_FILE)
+                .long("ftps-key-file")
+                .value_name("PEM_FILE")
+                .help("Sets the path to the private key file used for TLS security")
+                .env("UNFTP_FTPS_KEY_FILE")
```

Comment:
pelease add`.requires(FTPS_CERTS_FILE)` so `clap` can ensure that both parameters are present on startup.

Reply:
Good one. Will add